.drone.yml

```diff
@@ -0,0 +1,24 @@
+---
+clone:
+  git:
+    image: plugins/git:latest
+    pull: true
+
+pipeline:
+  test:
+    image: golang:1.12-buster
+    commands:
+      - export GO11MODULE=on
+      - make test
+
+  publish:
+    image: plugins/ecr
+    secrets: [ecr_access_key, ecr_secret_key, ecr_registry]
+    registry: ${ECR_REGISTRY}
+    repo: ${ECR_REGISTRY}devops/${DRONE_REPO_NAME}
```

Comment:
seems like you're missing a `/` before `devops`

Reply:
I thought so too.  However, drone is doing something magical here and adding a slash for me.